src/Component/FileImportButton.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import Button from "@material-ui/core/Button";
+
+type FileImportButtonProps = {
+  id: string;
+  buttonText: string;
+  fileType: string;

```

Comment:
Not a big deal, but since this maps directly to the `accepts` input attribute, I might just use the same terminology. A counter argument is that `fileType` is a little more clear, but a counter-counter argument is that `accepts` has specific syntax in terms of mime-type wildcards that the name `fileType` might not clearly imply to future consumers.

Reply:
Updated